packages/strapi-admin/admin/src/containers/Webhooks/ListView/tests/reducer.test.js

```diff
@@ -0,0 +1,211 @@
+import { fromJS } from 'immutable';
+import reducer from '../reducer';
+
+describe('Admin | containers | Webhooks | ListView | reducer', () => {
+  const initialState = fromJS({
+    webhooks: [],
+    webhooksToDelete: [],
+    webhookToDelete: null,
+  });
+
+  describe('Load webhooks', () => {
+    it('should update webhooks with received data', () => {
+      const state = initialState;
+      const receivedData = [
+        {
+          id: 1,
+          name: 'webhook 1',
+          url: 'http://localhost:5000',
+          headers: {},
+          events: ['entry.create', 'entry.update', 'entry.delete'],
+          isEnabled: true,
+        },
+        {
+          id: 2,
+          name: 'webhook 2',
+          url: 'http://localhost:4000',
+          headers: {},
+          events: ['media.create', 'media.update'],
+          isEnabled: false,
+        },
+      ];
+
+      const action = {
+        type: 'GET_DATA_SUCCEEDED',
+        data: receivedData,
+      };
+
+      const expectedState = state.set('webhooks', fromJS(receivedData));
+
+      expect(reducer(state, action)).toEqual(expectedState);
+    });
+  });
+
+  describe('Update webhook', () => {
+    it('should toggle isEnabled parameter', () => {
+      const state = initialState;
```

Comment:
You may need to initialise your initialState as your `webhooks` array is empty in your test.